apps/acrs/index.js

```diff
@@ -642,7 +642,9 @@ module.exports = {
       next: '/declaration'
     },
     '/declaration': {
-      fields: [],
+      fields: ['children-declaration'],
+      behaviours: [DeclarationBehaviour, SaveFormSession],
```

Comment:
I think you should move the Submit behaviour into here from the above '/confirm' step.